src/date-range-picker/index.tsx

```diff
@@ -204,13 +205,29 @@ const DateRangePicker = React.forwardRef(
       value = null;
     }
 
+    const i18n = useInternalI18n('date-range-picker');
+    const formatRelativeRange = i18n(
+      'i18nStrings.formatRelativeRange',
+      i18nStrings?.formatRelativeRange,
+      format =>
+        ({ amount, unit }) =>
+          format({ amount, unit })
+    );
+
+    if (!formatRelativeRange) {
```

Comment:
Let's add a check for development mode
```suggestion
  if(isDevelopment) {
    if (!formatRelativeRange) {
```
(with `import { isDevelopment } from '../internal/is-development';`)